src/App.js

```diff
@@ -73,8 +74,9 @@ class App extends Component {
           <Link to="/">Annotations</Link> | <Link to="/logout">Logout</Link>
           <Switch>
             <Route exact path="/">
-              <ImageAnnotation />
+              <AnnotationList />
             </Route>
+            <Route path = "/annotations/:id" children={<AnnotationPage />} /> 
```

Comment:
use [route url params](https://reacttraining.com/react-router/web/example/url-params ) to show annotation by id